drivers/dai/intel/ssp/dai-params-intel-ipc4.h

```diff
@@ -70,6 +70,35 @@ enum dai_intel_ipc4_connector_node_id_type {
 	dai_intel_ipc4_max_connector_node_id_type
 };
 
+/* Maximum number of devices */
+#define GTW_DMA_DEVICE_MAX_COUNT 16
+
+struct dma_device_stream_channel_map {
+	uint32_t device_address;
+	uint32_t channel_map;
+};
+
+struct dma_stream_channel_map {
+	uint32_t device_count;
+	struct dma_device_stream_channel_map map[GTW_DMA_DEVICE_MAX_COUNT];
+};
+
+struct ipc_dma_config {
+	uint8_t dma_method;		/**< DMA method */
+	uint8_t pre_allocated_by_host;	/**< 1 - DMA pre-allocated by host, otherwise to be
+					  *       allocated by DSP
+					  */
+	uint16_t rsvd;
+	uint32_t dma_channel_id;	/**< channel ID if DMA pre-allocated by host, otherwise
+					  *   invalid channel ID
+					  */
+	uint32_t stream_id;
+	/* DmaStreamChannelMap */
+	struct dma_stream_channel_map channel_map;
+	uint32_t dma_priv_config_size;
+	uint8_t  dma_priv_config[0];
+} __packed __aligned(4);
+
```

Comment:
this is already defined in src/include/sof/audio/ipc-config.h

Please don't add new definitions that are NOT SSP-specific anyways,